generated_files

```diff
@@ -9,6 +9,9 @@ mixer/adapter/kubernetes/config/config.pb.go
 mixer/adapter/list/config/config.pb.go
 mixer/adapter/memquota/config/config.pb.go
 mixer/adapter/prometheus/config/config.pb.go
+mixer/adapter/servicecontrol/config/config.pb.go
+mixer/adapter/servicecontrol/template/servicecontrolreport/go_default_library_handler.gen.go
+mixer/adapter/servicecontrol/template/servicecontrolreport/go_default_library_tmpl.pb.go
```

Comment:
it's weird genfiles under `mixer/adapter/svcctrl` are not removed from this file -- this is actually yet another breakage of `bazel_to_go.py`.

For now, please run `bazel clean`, and then `bazel build //...` and `bin/bazel_to_go.py`. That will remove those old file entries.